_posts/2020-10-01-Fortran-Newsletter-October-2020.md

```diff
@@ -0,0 +1,145 @@
+* [#175](https://github.com/fortran-lang/fpm/issues/175): Updated formatting of Markdown documents
+* [#174](https://github.com/fortran-lang/fpm/issues/174): Cache Haskell Stack build in CI
+* [#171](https://github.com/fortran-lang/fpm/issues/171): Increase test coverage of fpm manifest
+* [#170](https://github.com/fortran-lang/fpm/issues/170): Source parsing tests
+* [#163](https://github.com/fortran-lang/fpm/issues/163): Use different strategy to fetch git dependencies
+* [#162](https://github.com/fortran-lang/fpm/issues/162): Updated OS type identification
+* [#160](https://github.com/fortran-lang/fpm/issues/160): Add contributing guidelines
+(you can read them [here](https://github.com/fortran-lang/fpm/CONTRIBUTING.md))
+* [#157](https://github.com/fortran-lang/fpm/issues/157): Implement reading of fpm.toml
+* [#155](https://github.com/fortran-lang/fpm/issues/155): Internal dependencies and build backend
+
+
+Work in progress:
+
+* [#190](https://github.com/fortran-lang/fpm/issues/190) (WIP): Auto discovery of executables
+* [#189](https://github.com/fortran-lang/fpm/issues/189) (WIP): Implement `fpm new`
+* [#186](https://github.com/fortran-lang/fpm/issues/186) (WIP): Implement version string validation and comparison
+* [#182](https://github.com/fortran-lang/fpm/issues/182) (WIP): CLI interface implementation
+
+fpm is still in early development and we need as much help as we can get.
+Here's how you can help today:
+
+* Try to use it. Does it work? No? Let us know! Read the [fpm packaging guide](https://github.com/fortran-lang/fpm/blob/master/PACKAGING.md) to learn how to build your package with fpm.
+* Browse the [open issues](https://github.com/fortran-lang/fpm/issues) and see if you can help implement any fixes or features.
+* Adapt your Fortran package for fpm and submit it to the [Registry](https://github.com/fortran-lang/fpm-registry).
+* Improve the documentation.
+
+The short term goal of fpm is to make development and installation of Fortran packages with dependencies easier.
+Its long term goal is to build a rich and decentralized ecosystem of Fortran packages and create a healthy
+environment in which new open source Fortran projects are created and published with ease.
+
+##  Fortran benchmarks
+
+We created the [benchmarks repository](https://github.com/fortran-lang/benchmarks) with the goal to design and implement a comprehensive set of benchmarks.
+The benchmarks will aim to compare the performance of various Fortran compilers, as well as the performance of canonical algorithms implemented in Fortran and different languages.
+If you'd like to contribute in any way, be it the design, implementation, or testing of benchmarks, please join the ongoing discussion [here](https://github.com/fortran-lang/benchmarks/issues).
+
+## Compilers
+
+### GFortran
+
+TODO
+
+### Classic Flang
+
+TODO @gklimowicz
+
+### LLVM Flang
+
+TODO @gklimowicz
```

Comment:
```suggestion
Work continues on LLVM Flang, concentrating on semantics, lowering and runtime
sufficient to compile and run Fortran 77 programs.

In conjunction with the MLIR-based code from the _fir-dev_ fork (the Fortran
IR used for lowering), Flang can compile and run most F77 programs,
including the Fortran Compiler Validation Suite (FCVS).
We continue to work on refactoring necessary to upstream the _fir-dev_ fork into LLVM
flang proper.

Arm has contributed changes toward a full-fledged driver for flang.

AMD continues to add support for OpenMP semantics and lowering.

Valentin Clement continues to contribute parsing and semantics changes for
OpenACC support. This will be the topic of the next Flang Technical Community call
on Monday, October 5, 8:30 AM Pacific Time.

Michael Kruse continues to add support for building Flang on Windows with MSVC.
```